lib/hawkular/operations/operations_api.rb

```diff
@@ -105,7 +127,8 @@ def connect
         client.on(:message, once: true) do |msg|
           parsed_message = msg.data.to_msg_hash
 
-          logger.log("Sent WebSocket message: #{parsed_message}")
+          # Not working with websocket-vcr
```

Comment:
Due to [the way the callback is invoked](https://github.com/shokai/event_emitter/blob/e7ad10c0755c0427d69c3653caac4c2a222adce4/lib/event_emitter/emitter.rb#L53), `logger` becomes out of scope and undefined.